extensions/google-chrome/src/util/index.ts

```diff
@@ -81,6 +82,42 @@ export const getBookmarks = async (profile?: string): Promise<HistoryEntry[]> =>
     throw new Error(NO_BOOKMARKS_MESSAGE);
   }
 
-  const fileBuffer = await fs.promises.readFile(bookmarksFilePath, { encoding: "utf-8" });
+  const fileBuffer = await fs.promises.readFile(bookmarksFilePath, {
+    encoding: "utf-8",
+  });
   return extractBookmarks(JSON.parse(fileBuffer));
 };
+
+const fuzzyDefaultPreferenceName = "default";
+const fuzzyDefaultThreshold = 0;
+const fuzzySearchPreferencesMap = new Map<string, number>([
+  [fuzzyDefaultPreferenceName, fuzzyDefaultThreshold],
+  ["balanced", 0.4],
+  ["loose", 0.7],
+]);
+
+export function filterElements<T>(items: T[], query: string | undefined, keys: string[], threshold: number) {
+  if (!query) {
+    return items;
+  }
+
+  if (threshold) {
```

Comment:
Why this conditional? Wouldn't  searching with a threshold of 0 have the same effect?